test/test_sorted_containers.jl

```diff
@@ -32,6 +38,17 @@ function my_primes(N)
     p
 end
 
+function remove_spaces(s::String)
+    b = Vector{UInt8}()
+    for c in s
+        if !isspace(c)
+            push!(b,UInt8(c))
+        end
+    end
+    String(b)
+end
+    
```

Comment:
```suggestion
remove_spaces(s::String) = replace(s, re"\s+"=>"")    
```

Reply:
I found out that it is actually `r"\s+"` rather than `re"\s+"`, but aside from that, this is done.